### DISCUSSION TITLE
Videos are broken?

### DISCUSSION THREAD

**Discussion by Sebastian1175** [Avnsx/fansly#115] [Q&A] [answered]
Hi!
Love youre software, but i ran into some issues with downloaded Videos. Most of them are not playing or have fragments.
Is it a known problem? I Use the 0.4 Software
Thanks!

**Comment by Avnsx** [comment]
Yes it's a known issue, please download the [latest python version](https://github.com/Avnsx/fansly-downloader#python-version-requirements) of the repository and let me know if those changes, solve your issue.

**Reply by Sebastian1175** [reply]
Just wanted to let you know that the problem with the Python Version is the same.

Edit: I used  Python 3.11.4

**Reply by Avnsx** [reply]
What graphics card, cpu and how much ram do you have?

**Reply by Sebastian1175** [reply]
AMD A10-7300 Radeon R6 Graphics, 10 Compute Cores 4C+6G    1.90 GHz
RAM	8,00 GB 

It doesent effect all Videos, just specific. I tried to download them seperatly (Cofig data) but it is always the same result.

**Reply by Avnsx** [reply, marked as answer]
Seems like you've a integrated low end graphics card.

After version 0.4 videos need to be converted from m3u8 to mp4, which requires a average or better graphics card.

Also you only have 8 gb RAM. All videos are loaded into RAM now before being coverted to mp4, if the video you're trying to download is bigger in file size it might be possible that you're running out of RAM.

I might integrate a way to just revert to 0.3.5 version video downloads, that would mean you wouldn't be able to download the highest quality video though and might end up with only being able to download 720p versions of videos (same as in version 0.3.5). But hey atleast you can then download videos again, I guess.

**Reply by Sebastian1175** [reply]
Yes, its a pretty old and crappy Laptop. I appreciate your help.

**Comment by Sebastian1175** [comment]
Im using the exe Version of the Downloader. So you mean it works better
when I use Python instead?

Mika C. ***@***.***> schrieb am Sa., 24. Juni 2023, 20:05:

**Reply by Avnsx** [reply]
The python version is on 0.4.1, which contains fixes for the issue you're mentioning.

The executable version is still on v0.4

**Comment by Sebastian1175** [comment]
Ok. Thank you!
I will try to setup it soon.
Keep up the good work!
Best Regards, Sebastian.

Mika C. ***@***.***> schrieb am Sa., 24. Juni 2023, 20:09:
